registration/provision.py

```diff
@@ -93,6 +93,7 @@ def _provision_instance(sender, **kwargs):
 
         if settings.PROD_APPSERVER_FAIL_EMAILS:
             application.instance.provisioning_failure_notification_emails = settings.PROD_APPSERVER_FAIL_EMAILS
+            application.instance.additional_monitoring_emails = settings.PROD_APPSERVER_FAIL_EMAILS
```

Comment:
@swalladge, afaik, the monitoring SLA is applicable only for paying customers - P&T and institutional clients and that is why we add `urgent@` manually to those instances.

Reply:
@lgp171188 This is the `production_instance_factory`, so it's fine. If a developer ends up spawning an appserver that fails, it's his responsibility of cleaning it up/addressing the page. 
I'd prefer keep this explicit mechanism instead of having to manually set the monitoring.

Reply:
Btw @swalladge, take a look at this: https://github.com/open-craft/opencraft/pull/489 (your previous PR here).
Do we really need this change here? I assumed we needed to set the emails manually, but for P&T, they are already added, so no problem.

Reply:
@giovannicimolin according to the original requirements, yes we did need this change.

> Doing this because we only want to auto set PROD_APPSERVER_FAIL_EMAILS for end user instances.

You can check SE-1832, but I believe the reason to move things to here, is that now they will be autoset up for new instances that are set up through the betatest registration process (our standard documented process for setting up client and maintained instances).

Reply:
Also, provisioning_failure_notification_emails != additional_monitoring_emails, so I'm not sure what you're asking. :thinking:

Reply:
Yup, I think I've mixed up those two.